libs/payments/stripe/src/lib/card.factory.ts

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { faker } from '@faker-js/faker';
+import { Stripe } from 'stripe';
+
+export const CardFactory = (override: Partial<Stripe.Card>): Stripe.Card => ({
+  id: 'card',
+  object: 'card',
+  address_city: 'Anytown',
+  address_country: 'US',
+  address_line1: null,
+  address_line1_check: null,
+  address_line2: null,
+  address_state: faker.location.state({ abbreviated: true }),
+  address_zip: faker.location.zipCode(),
+  address_zip_check: null,
+  brand: 'Visa',
+  country: 'US',
+  currency: 'USD',
+  cvc_check: null,
+  dynamic_last4: null,
+  exp_month: faker.number.int({ min: 1, max: 12 }),
+  exp_year: faker.date.future({ years: 10 }).getFullYear(),
+  funding: 'credit',
+  last4: '4242',
```

Comment:
Likely sub this with https://fakerjs.dev/api/finance.html#creditcardnumber but slice the last 4 digits